### PR TITLE
Update 18

### PR DESCRIPTION
**Pull request by Rainb0wKey**
Hello!

### CODE REVIEW

File: src/bdscript/customID.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
 # $customID
-Can only be used with the [$onInteraction](/src/callbacks/onInteraction.md) callback. Returns the custom ID of this interaction.
+Can only be used with the [`$onInteraction`](/src/callbacks/onInteractionComplex.md) callback. Returns the custom ID of this interaction.
```
Comment (noituri) [inline src/bdscript/customID.md]: Please use relative url instead

```suggestion
Can only be used with the [`$onInteraction`](../callbacks/onInteractionComplex.md) callback. Returns the custom ID of this interaction.
```

File: src/bdscript/date.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # $date
 Returns the current date.
-> 🧙‍♂️ You can use [$time](https://nilpointer-software.github.io/bdfd-wiki/bdscript/time.html) to change the timezone.
+> You can use [`$time[]`](/src/bdscript/time.md) to change the timezone.
```
Comment (noituri) [inline src/bdscript/date.md]:
```suggestion
> You can use [`$time[]`](./time.md) to change the timezone.
```

**Review by noituri — CHANGES_REQUESTED** [review]

**Comment by Rainb0wKey** [comment]
Thanks

**Review by MineBartekSA — APPROVED** [review]
Thank you for your contribution

**Review by noituri — APPROVED** [review]

**Pull request merged**